KUBERNETES.md

```diff
@@ -0,0 +1,231 @@
+## Running: Kubernetes
```

Comment:
@longshorej, it's not clear to me how this readme will be linked to the other one. It would be nice to have the intro of the primary readme explain that there are instructions for deploying the sample to Kubernetes and then link to this?

Reply:
Yeah - there's a link from `README.md` to `KUBERNETES.md`, see here: https://github.com/longshorej/online-auction-scala-orchestration-example/blob/8f40fd4064d8e8797a27ba983fb575c70ea9ecd7/README.md